spec/README.md

```diff
@@ -66,3 +67,23 @@ A **_term_** looks like this when it is defined in this specification.
 A reference to a _term_ looks like this.
 
 > Examples are non-normative and styled like this.
+
+### Stability Policy
+
+Updates to this specification MUST NOT change the formatting or other behaviour
+of valid messages written for earlier versions of this specification
+that only use functions and expression attributes defined in this specification.
+Later specification versions MAY make previously invalid messages valid.
+
+Updates to this specification MUST NOT introduce message syntax that,
+when parsed according to earlier versions of this specification,
+would produce syntax or data model errors.
+Such messages MAY produce errors when formatted
+according to an earlier version of this specification.
+
+Later versions of this specification MAY define new function names,
+function option names, and expression attribute names with new meanings.
+Such names defined in this specification MUST consist only of
+characters in the ranges a-z, A-Z, and 0-9.
+Users defining custom names SHOULD include at least one character outside these ranges
```

Comment:
Why not specify/suggest that custom names (i.e. private names) be prefixed with '`_`' or some other traditional indicator of intent? The "some character outside of [a-zA-Z0-9] somewhere in the name" seems a bit squishy if technically verifiable.

Reply:
I would be open to including a specific suggestion like that, but it probably ought to allow for namespacing like `moz-platform` or `com.google.number`. If we ever define some more structure within the _name_ rule then that ought to be obviously followed.

The rule should not be hard, though, as that would prevent e.g. customizing existing formatters with custom options.

Reply:
This guidance should be a note, not part of the stability guarantee directly... or be in the specification body or user guide. That is, this probably should be mentioned in [syntax#function](https://github.com/unicode-org/message-format-wg/blob/main/spec/syntax.md#function) etc.

Reply:
Put the last sentence into a note, but did not at least yet add a specific suggestion.